Bottom: 55752b017222f0e166f19c451c5175c0e4be5088
Top:    5ff79c3c4bdccfb0b1526d50e9310b498c74acd0
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-01-09 18:57:09 +0530

Refresh of sched-fair-tune-task-wake-up

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 76e1a379e6d3..943065096ff9 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5892,7 +5892,7 @@ static int select_idle_cpu(struct task_struct *p, struct sched_domain *sd, int t
 
 static inline bool is_background_task(struct task_struct *p)
 {
-	if (lenient_latency(p) && !is_zealous(task_util(p)))
+	if (task_latency_lenient(p) && !is_zealous(task_util(p)))
 		return true;
 
 	return false;
